app/routes/search.js

```diff
@@ -62,6 +68,18 @@ export default Route.extend(
           });
         });
 
+        if (!this.get('fastboot.isFastBoot')) {
+          this.adsContextService.getAdsContext()
+            .then((adsContext) => {
+              if (this.get('ads.module.isLoaded')) {
+                this.get('ads').setupAdsContext(adsContext);
+              } else {
+                this.get('ads.module').init(adsContext);
```

Comment:
Why do we use this `this.get` syntax instead of regular like `this.ads.module.init(adsContext)`? It is mixed up in the entire project. Even in this very function we call `this.adsContextService.getAdsContext()` regularly, but `this.get('ads.module').init(adsContext)` through this `this.get` syntax.